.github/workflows/node.js.yml

```diff
@@ -10,16 +10,52 @@ on:
         branches: [master]
 
 jobs:
+    clone_vector_skin:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Clone vector skin repository"
+              run: |
+                  git clone "https://gerrit.wikimedia.org/r/mediawiki/skins/Vector" skins/Vector
+
+    mediawiki_docker:
+        env:
+            MW_DOCKER_PORT: 8080
+            MW_SCRIPT_PATH: /w
+            MW_SERVER: http://localhost:8080
+            MEDIAWIKI_USER: Admin
+            MEDIAWIKI_PASSWORD: dockerpass
+            XDEBUG_CONFIG: ""
+        runs-on: ubuntu-latest
+        needs: clone_vector_skin
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Add docker-compose.override.yml"
+              run: | 
+                  $(pwd)/.github/workflows/docker_compose_override.sh $(id -u) $(id -g)
```

Comment:
I believe you are missing out on two environment variables here:

https://github.com/hariclerry/mediawiki/blob/master/DEVELOPERS.md#linux-users

Reply:
Please also create a .env file as per the instructions